homeassistant/components/directv/config_flow.py

```diff
@@ -76,8 +75,7 @@ def _show_form(self, errors: Optional[Dict] = None) -> Dict[str, Any]:
             return self._show_form(errors)
         except Exception:  # pylint: disable=broad-except
             _LOGGER.exception("Unexpected exception")
-            errors["base"] = ERROR_UNKNOWN
-            return self._show_form(errors)
+            return self.async_abort(reason=ERROR_UNKNOWN)
```

Comment:
We need to move the unknown error string to abort in `strings.json`.

Reply:
https://github.com/home-assistant/core/pull/32807